src/main/java/com/contrastsecurity/sdk/scan/CodeArtifactInner.java

```diff
@@ -0,0 +1,52 @@
+package com.contrastsecurity.sdk.scan;
+
+import com.google.auto.value.AutoValue;
+import java.time.Instant;
+
+/** Value type that describes the code artifact structure returned by the Contrast API. */
+@AutoValue
+abstract class CodeArtifactInner {
```

Comment:
I'm not crazy about this naming convention, but I think `Inner` is better than `Value` and both are certainly better than `Form`. I don't have any suggestions for the time being - and i like that this pattern gets us around some GSON hiccups and ugliness